pycbc/waveform/parameters.py

```diff
@@ -205,39 +205,39 @@ def docstr(self, prefix='', include_label=True):
 q = Parameter("q",
                 dtype=float, label=r"$q$",
                 description="The mass ratio, m1/m2, where m1 >= m2.")
-m_p = Parameter("m_p",
-                dtype=float, label=r"$m_{\mathrm{pr}}$",
+primary_mass = Parameter("primary_mass",
```

Comment:
I assume it still means primary mass though, and *not* mass1 which could be the lighter one? That's my confusion, I guess.